tests/slow/ParallelRestoreOldBackupCorrectnessAtomicOp.txt

```diff
@@ -25,6 +25,8 @@ testTitle=BackupAndParallelRestoreWithAtomicOp
     clearAfterTest=false
     simBackupAgents=BackupToFile
     backupRangesCount=-1
+;    use old backup
```

Comment:
I see you created old backup and new backup for other tests. How about add one of new backup for this test?

Reply:
Oh, I renamed the file and didn't add it into git. 
The new commits add them and resolve the comments.